packages/core/admin/admin/src/content-manager/components/AttributeFilter/index.js

```diff
@@ -18,7 +20,7 @@ const AttributeFilter = ({ contentType, slug, metadatas }) => {
 
     return {
       name,
-      metadatas: { label },
+      metadatas: { label: formatMessage({ id: label }) },
```

Comment:
most attribute filters are dynamic since they are based on content-type label fields (set in the content-type builder) - I'm not sure how this fix would help since they won't exist in translation files:

<img width="1072" alt="image" src="https://user-images.githubusercontent.com/71838159/182183234-63f56608-2f06-48fe-8097-84dec7319aa8.png">

could you explain the use case / issue you wanted to fix? 🙂

Reply:
As said in document: [Extending translations](https://docs.strapi.io/developer-docs/latest/development/admin-customization.html#extending-translations), we can "Customize the label of the Content Manager table" like this:
```
export default {
  config: {
    translations: {
      fr: {
        // Customize the label of the Content Manager table.
        Id: 'ID french',
      },
    },
  },
};
```

So localized filter labels will improve user experience for non-english user.

Default:
<img width="330" alt="Screen Shot 2022-08-03 at 9 13 16 AM" src="https://user-images.githubusercontent.com/22420/182503137-668e1b70-0319-47d9-8759-f9c55968b9bc.png">

Chinese simplified (zh-Hans)
<img width="312" alt="Screen Shot 2022-08-03 at 9 13 37 AM" src="https://user-images.githubusercontent.com/22420/182503333-b53931b5-9f02-4a8b-ad74-167e93bcca7f.png">

Reply:
interesting, didn't think about this translation extension use case!
just one thing then, could you pass the label as defaultMessage too please?
```
metadatas: { label: formatMessage({ id: label, defaultMessage: label }) },
```